polls/resource.py

```diff
@@ -72,6 +73,11 @@ def get(self, request, *args, **kwargs):
             if len(links) > 0:
                 response['Link'] = ', '.join(links)
 
+        if str(content_type) != 'application/vnd.siren+json':
+            # Add an Allow header
+            methods = ['HEAD', 'GET'] + map(lambda a: a.method, self.get_actions().values())
+            response['allow'] = ', '.join(methods)
```

Comment:
Yes, `response['header name'] = 'value'` sets a header in the response in Django.